web/src/elements/forms/SearchSelect/index.ts

```diff
@@ -0,0 +1,4 @@
+import { SearchSelect } from "./ak-search-select";
```

Comment:
I think we should probably rip the bandaid off and update all the imports to use the new file instead of keeping this "wrapper" around